### PR TITLE
3VM: create pallet account manager

### PR DESCRIPTION
**Pull request by AwesomeIbex**
## Summary
This adds an MVP of `pallet-account-manager` that handles deposits, finalisation(from circuit), issuance and a very very basic splitting.

This pallet will also be the provider for `EscrowTrait` in the future. If we even keep it, since it screws up the `BalanceOf` type, i might remove it when i pick up the follow-on tickets for this epic.

This also updates the author on registry accounts to be more ergonomic.

At present, we have no extrinsics for this pallet, although it might be necessary to have sudo extrinsics to demonstrate functionality. << im gonna add these sudo extrinsics right now :)

Also formats tomls.

## Checklist
- [ ] insert any notable changes, in order of commit

## Please check that your PR completes the requirements as follows
- [x] Tests have been added for bug fixes/features
- [x] Acceptance criteria from the issue has been completed

### CODE REVIEW

**Comment by AwesomeIbex** [comment]
TODO: fix a test that now no long breaks since we dont control execution id

File: pallets/account-manager/src/manager.rs [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        // TODO: these need to be joined or handle failure, maybe on_initialize retry a queue of failures after reserving
+        pay_split(payee_split, item.payee())?;
+        pay_split(recipient_split, item.recipient())?;
+
+        Ok(())
+    }
+}
+
+#[cfg(test)]
+mod tests {
```
Comment (beqaabu) [inline pallets/account-manager/src/manager.rs:105]: Maybe move these tests to the tests.rs file?

**Review by beqaabu — APPROVED** [review]
Really cool! :clinking_glasses:

File: pallets/account-manager/src/manager.rs (hunk 1 above)
Comment (AwesomeIbex) [reply pallets/account-manager/src/manager.rs:105]: Ah, the tests.rs is more for integration-like tests. These are unit tests and normally stay in the module

**Pull request merged**